cuda_phase-retrieval/documentation/doc_config.py

```diff
@@ -0,0 +1,2656 @@
+# Doxyfile 1.9.1
```

Comment:
Please rename the file to `Doxyfile`. It is the default name and `doxygen` can find it without extra arguments:

```bash
$ cd cuda_phase-retrieval/documentation
$ doxygen
warning: Tag 'CLANG_ASSISTED_PARSING' at line 1161 of file 'Doxyfile' belongs to an option that was not enabled at compile time.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u", or recompile doxygen with this feature enabled.
warning: Tag 'CLANG_ADD_INC_PATHS' at line 1167 of file 'Doxyfile' belongs to an option that was not enabled at compile time.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u", or recompile doxygen with this feature enabled.
warning: Tag 'CLANG_OPTIONS' at line 1175 of file 'Doxyfile' belongs to an option that was not enabled at compile time.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u", or recompile doxygen with this feature enabled.
warning: Tag 'CLANG_DATABASE_PATH' at line 1188 of file 'Doxyfile' belongs to an option that was not enabled at compile time.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u", or recompile doxygen with this feature enabled.
Doxygen version used: 1.9.1 (0b26ee49f10a0c11c1a58baa2f9c51202cad4e78*)
Searching for include files...
Searching for example files...
Searching for images...
#...
```